app.rb

```diff
@@ -30,12 +33,88 @@ def menu
       input = gets.chomp.to_i
       case input
       when 1..6 then options(input)
-      when 7 then puts 'Thank you for using our app.'
+      when 7
+        save_data
+        puts 'Thank you for using our app.'
       end
       break if input == 7
     end
   end
 
+  def read_data
+    books = File.read('data_files/books.json')
+    people = File.read('data_files/people.json')
```

Comment:
- [ ] You need to handle the case where the storage files do not exist and the app is started. You seem to have only handled the case where the rental.json file does not exist - which is good. But for the books and people JSON files, I get this error when I start the app. Kindly fix that.

  You should ensure no error is presented when the app is started even if the files do not exist - and after the app is exited the storage files should be created and the new data saved.

  ![image](https://user-images.githubusercontent.com/81468675/210536289-8b9b8990-15b5-48a5-bce5-54080f48064d.png)